app/routes/surveys/index.js

```diff
@@ -44,7 +44,6 @@ const surveysRoute = ({ match }: { match: { path: string } }) => (
           Component={SurveyDetailRoute}
         />
         <RouteWrapper
-          exact
```

Comment:
Why did you remove `exact` here? If the idea is that the above route matches first, I still like the explicit handling to avoid unexpected behaviour.

Reply:
Thought it had something to do with urls with query params not being matched, but it turned out to be the templates not being in the backend fixtures. Will add it back in